README.md

```diff
@@ -105,6 +105,8 @@ A curated list of awesome MATLAB toolboxes, applications, software and resources
 * [saveFigure](https://github.com/djoshea/matlab-save-figure) - exports MATLAB® figures to PDF, SVG, EPS and PNG with improved support for transparancy.
 * [hq-matlab-figs](http://dgleich.github.io/hq-matlab-figs/) - Creating high-quality graphics in MATLAB for papers and presentations.
 * [antijet](https://git.io/antijet) - An alternative to the problematic "jet"-colormap.
+* [octave kernel](https://github.com/calysto/octave_kernel) - to create IPython / Jupyter notebooks.
```

Comment:
1. Is there a reason why you include the Octave kernel and not the [MATLAB one](https://github.com/calysto/matlab_kernel)?
2. Also, it is not really graphics-related. I would rather classify this as "interfacing with other languages".
3. I think the description is not correct. Those do not allow to create such notebooks. They allow to run code in Octave/MATLAB from within Jupyter (previously IPython) notebooks and allow to transfer data and graphics between Jupyter and MATLAB/Octave.